admin/app/controllers/authenticated/certifications/sessions/info/index.js

```diff
@@ -7,7 +7,7 @@ import { statusToDisplayName } from '../../../../../models/session';
 export default Controller.extend({
 
   sessionInfoService: service(),
-  notifications: service('notification-messages'),
+  notifications: service(),
```

Comment:
Suggestion: rename branch name from tech-upgrade-ember-315-on-pix-admin to pa-142-infra-bump-ember-from-??-to-3.15